tools/accuracy_checker/accuracy_checker/data_analyzer/detection_analyzer.py

```diff
@@ -44,31 +45,50 @@ def analyze(self, result: list, meta, count_objects=True):
             counter.update(data.labels)
 
         if count_objects:
-            self.object_count(result)
+            data_analyze['annotations_size'] = self.object_count(result)
 
         print_info('Total boxes {}'.format(all_boxes))
+        data_analyze['all_boxes'] = all_boxes
         label_map = meta.get('label_map', {})
 
         for key in counter:
             if key in label_map:
                 print_info('{name}: {value}'.format(name=label_map[key], value=counter[key]))
+                data_analyze[label_map[key]] = counter[key]
             else:
                 print_info('class_{key}: {value}'.format(key=key, value=counter[key]))
+                data_analyze['class_' + key] = counter[key]
 
         if size > 0:
+            avg_num_diff_objects = diff_objects/size
+            avg_num_all_objects = all_boxes/size
+            avg_width = width/all_boxes
+            avg_height = height/all_boxes
+
             print_info(
-                'Average number of difficult objects (boxes) per image: {average}'.format(average=diff_objects/size)
+                'Average number of difficult objects (boxes) per image: {average}'.format(average=avg_num_diff_objects)
             )
             print_info(
-                'Average number of objects (boxes) per image: {average}'.format(average=all_boxes/size)
+                'Average number of objects (boxes) per image: {average}'.format(average=avg_num_all_objects)
             )
             print_info(
                 'Average size detection object: width: {width}, '
-                'height: {height}'.format(width=width/all_boxes, height=height/all_boxes)
+                'height: {height}'.format(width=avg_width, height=avg_height)
             )
 
+            data_analyze['average_num_diff_objects'] = avg_num_diff_objects
```

Comment:
```suggestion
            data_analyze['average_num_ignored_objects'] = avg_num_diff_objects
```
or
```suggestion
            data_analyze['average_num_diff_objects'] = avg_num_diff_objects
```
```suggestion
            data_analyze['average_num_difficult_objects'] = avg_num_diff_objects
```

field name unclear (diff usually used in meaning difference, but here meant difficult)